testdata/docker/Makefile

```diff
@@ -0,0 +1,31 @@
+# Makefile to build and push the `cilium/ebpf` llvm builder Docker image.
+DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+REPO := $(shell cat ${DIR}/REPO)
+VERSION := $(shell cat ${DIR}/VERSION)
+
+ifndef REPO
+$(error REPO file not present in Makefile directory)
+endif
+ifndef VERSION
+$(error VERSION file not present in Makefile directory)
+endif
+
+DATE := $(shell date +'%d%m%Y')
+TAG := ${VERSION}-${DATE}
+
+build: ## Build and tag the Docker image in REPO with VERSION.
+	docker build -f "${DIR}/Dockerfile.builder" "${DIR}" \
+		-t ${REPO}:${TAG} \
+		-t ${REPO}:${VERSION}
+
+push: ## Push the Docker image to the repo specified in REPO.
+	docker push	${REPO}:${TAG}
+	docker push ${REPO}:${VERSION}
+
+help: ## Display this help.
+	@grep -E '^[a-zA-Z_-]+:.*## .*$$' $(MAKEFILE_LIST) | \
```

Comment:
I don't understand what this does, is it better than just looking at the Makefile?

Reply:
In this case, probably not, since it's only build/push and there's a readme now. It's a convenience, nothing more, I include it by default.